mobx/lib/model/layout.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:mobx/mobx.dart';
+import 'package:mobx_sample/model/todo_list.dart';
+
+part 'layout.g.dart';
+
+class LayoutStore = _LayoutStore with _$LayoutStore;
```

Comment:
Frankly there is no need to create complete store here as the only observable is a `activeTab`. I would suggest creating a top-level `TodoManagerStore` that contains the lists/stats and activeTab props and sub-stores.

Reply:
@pavanpodila  So basically you want to create a LayoutManagerStore that will keep `activeTab` prop and an instance of `TodoListStore` right?

Reply:
I have made the changes to the store architecture, please have a look.